python/hail/dataset.py

```diff
@@ -1097,16 +1120,19 @@ def concordance(self, right):
                 VariantDataset(self.hc, result._2()))
 
     def count(self, genotypes=False):
-        """Return number of samples, variants and genotypes.
+        """Gets number of samples, variants and genotypes in this vds as a dictionary with the fields ``"nSamples"``, ``"nVariants"``, and ``"nGenotypes"``.
```

Comment:
I'd change to: "Returns number of samples, variants and genotypes as a Python dictionary with the fields..."

And if it's a Python dictionary that the fields are strings with single quotes.